browserslist.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Precise browser targeting reduces the amount of transpilation we do.
+ * Precise browser targeting reduces the amount of transpiration we do.
```

Comment:
This is not a typo.
Transpiling is a specific term for taking source code written in one language and transforming into another language that has a similar level of abstraction.